reviews/review-03+2020-11-30.md

```diff
@@ -0,0 +1,169 @@
+convinced the current order is the best possible one. Perhaps storyboarding the
+entire book would help the authors discover opportunities to reorganize. The
+approach I suggest is a stylistic preference and might mean making chapters less
+monolithic, returning to topics as they are needed.
+
+b. Sec 1.1: Mentions "three" related concepts; why not give sustainability its
+own bullet point? I love your definition.
+
+c. Sec 1.1: Grammatical rewrite needed: "Nobody argues that research should be
+irreproducible or unsustainable, but "not against it" and actively supporting it
+are very different things. Academia doesn't yet know reward people for writing
+useful software, so while you may be thanked, the effort you put in may not
+translate into job security or decent pay."
+
+d. Sec 1.2: I assume the audience are learner personas---perhaps ones that will
+be used throughout the book. This needs explanation because it is jarring to
+just have three "audience (members? personas?)" just pop into existence on the
+page. Why not explain the use of the personas and how it benefits the reader to
+relate to them at this point in the text?
+
+e. Sec 1.2.1: With the exception of your clearly stating Python 3 is needed,
+might it be better to specify versions that were used in the creation of the
+book examples? I think it is fine to allow learners freedom, especially since at
+this point, they would not be novices and probably have their own preferences. I
+might still want these specifics (especially if I were teaching this material)
+and those could be specified in detail in the GitHub repo accompanying the
+book. I don't seem to find Appendix E in my PDF version of the manuscript.
+
+f. Sec 1.3: I might add a sentence or two generalizing the Zipf's law
+example. If you want to "sell" the book, I think I would emphasize how looking
+for patterns in data is common to most scientific analyses and so you have
+chosen a straightforward example that will allow you to teach the concrete
+objectives you state. Rather than Syllabus, maybe something more direct---"What
+you will learn."
+
+g. Given what I've read up the Chapter 2 on the Unix shell, I am under the
+assumption that the audience of this book is already using Python at least an
+intermediate level. I'd like to believe that this level of user at least knows
+how to manipulate Linux directories. Everything in this chapter felt like
+prerequisite, so perhaps it should be an appendix?
+
+h. I think Chapter 3 content was more appropriate for an intermediate Python
+user who did not do much with the shell. I might make this 'the' Unix chapter
+with a little bit of introductory context as to why the content will be
+important for RSEs to be effective in that role.
+
+i. Should Sec 7.6 discuss using GitHub issues templates to create more effective
+bug reports.
+
+j. I think Docker is an essential reproducibility tool and feel that a chapter
+or section on the subject would be useful.
```

Comment:
We have a call-out box about Docker in the provenance chapter, with a citation to more information. I think that's probably sufficient.